docs/packages/dev-workflow.md

```diff
@@ -2,93 +2,120 @@
+      OCI_REGISTRY="****" \
+      PACKAGE_NAME="readiness" \
+      PACKAGE_REPOSITORY="management" \
+      PACKAGE_VERSION="dev.0.0" \
+      PACKAGE_SUB_VERSION="1" \
+      REGISTRY_USERNAME="****" \
+      REGISTRY_PASSWORD="****" \
+      REGISTRY_SERVER="****" \
+      make package-bundle-generate -f build-tooling.mk
    ```
 
-5. Build package repo bundle
+4. Build package repo bundle [Optional]
 
    After building the package bundles, it is time to build package repository bundle, run the below make
    target:
 
    ```shell
-      PACKAGE_REPOSITORY=management OCI_REGISTRY=${REGISTRY} make package-repo-bundle
+      PACKAGE_REPOSITORY=management OCI_REGISTRY="****" make package-repo-bundle
    ```
 
-6. Push package bundles
+5. Push package bundles 
 
    After the package bundles are generated, now it's time to push them to an OCI registry, so that they can be consumed.
    Run the below make target to push all the package bundles in the specified package repository:
 
    ```shell
-      PACKAGE_REPOSITORY=management OCI_REGISTRY=${REGISTRY} make push-all-package-bundles
+      OCI_REGISTRY="****" \
+      PACKAGE_REPOSITORY="management" \
+      PACKAGE_VERSION="dev.0.0" \
+      PACKAGE_SUB_VERSION="1" \
+      REGISTRY_USERNAME="****" \
+      REGISTRY_PASSWORD="****" \
+      REGISTRY_SERVER="****" \
+      make package-bundle-push-all -f build-tooling.mk
    ```
 
-   If you are interested in pushing only a few package bundles, you could do that by running
+   If you are interested in pushing only a specific package bundle, you could do that by running
 
    ```shell
-      OCI_REGISTRY=${REGISTRY} make push-package-bundles ${PACKAGE_BUNDLES}
+      OCI_REGISTRY="****" \
+      PACKAGE_NAME="readiness" \
+      PACKAGE_REPOSITORY="management" \
+      PACKAGE_VERSION="dev.0.0" \
+      PACKAGE_SUB_VERSION="1" \
+      REGISTRY_USERNAME="****" \
+      REGISTRY_PASSWORD="****" \
+      REGISTRY_SERVER="****" \
+      make package-bundle-push -f build-tooling.mk
    ```
 
-   PACKAGE_BUNDLES variable should be comma-separated values and must not contain spaces.
-   Example: PACKAGE_BUNDLES=featuregates,core-management-plugins
-
-7. Push repo bundle
+6. Push repo bundle [Optional]
 
    To push the generated repo bundle to an OCI registry, run:
 
    ```shell
-      PACKAGE_REPOSITORY=management OCI_REGISTRY=${REGISTRY} make push-package-repo-bundle
+      PACKAGE_REPOSITORY=management OCI_REGISTRY="****" make repo-bundle-push -f build-tooling.mk
    ```
 
+----
+
 Follow the below steps to test the artifacts that are generated in previous steps on your local cluster
 
-**Note**: A local cluster can be any k8s cluster with `kapp-controller` installed in it.
+> **Note**: A local cluster can be any k8s cluster. There are 2 ways to install the packages
+
+#### To install on a cluster without `kapp-controller` installed in it.
+
+1. Download the package bundle. The sha tag can be noted from the registry where package was published.
```

Comment:
I think the instructions should be to create the PackageRepository CR first, which makes the packages available in the cluster, and then to install a particular package they are interested in by creating the PackageInstall CR

or

pull and extract the repo bundle, and create the Package CR and the PackageInstall CR

Reply:
Updated